### DISCUSSION TITLE
MudDataGrid

### DISCUSSION THREAD

**Discussion by laurolapajunior** [balta-io/forum#615] [Tira Dúvidas] [answered]
Boa noite, como faço para deixar as rows do MudDataGrid com um height menor? por padrão ele usa classe .mud-table-cell que tem um padding de 16px, tentei até alterar mas não consegui.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @laurolapajunior, como vai?

Não sei se tem como alterar isso nos temas do MudBlazor, acho que é fixo, porém, via CSS podemos tudo hahahaha
Tentou isso 👇
```css
.mud-table-cell { padding: 32px !important; }
```

**Reply by laurolapajunior** [reply]
pior que tentei, criei um css especifico da pagina, e fiz isso, mas não mudou nada.

**Reply by andrebaltieri** [reply]
Mas qual CSS tentou criar, em qual página?

**Comment by laurolapajunior** [comment]
Tenho uma página List.razor criei List.razor.css e coloquei esse código que
vc passou, mas não refletiu na pagina

Em qui., 11 de jul. de 2024 13:57, Andre Baltieri ***@***.***>
escreveu:

**Reply by andrebaltieri** [reply]
Bom dia, @laurolapajunior, como vai? 
Não veio o código!

**Comment by laurolapajunior** [comment]
boa noite, nem enviei o código, mas entendi o problema, pelo que vi, a
questão de criar o css para para página, só funciona se for componente,
sem o @page no inicio, caso contrário não funciona, é isso mesmo?  e achei
um propriedade
Dense="true" do datagrid que diminui os height das linhas, na verdade
diminui o padding.

att,

Em sex., 12 de jul. de 2024 às 08:03, Andre Baltieri <
***@***.***> escreveu:

> Bom dia, @laurolapajunior <https://github.com/laurolapajunior>, como vai?
> Não veio o código!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/615#discussioncomment-10030629>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZUHEXK6DPCATKGB6432NDZL6ZXPAVCNFSM6AAAAABKT5HKPWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBTGA3DEOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Lauro
Batista Lapa Junior

Analista de Sistemas
3L Soluções
***@***.***
(44) 98850-2722
<https://goo.gl/qxcoqu> <https://goo.gl/JksoVf> <https://goo.gl/dL8XRG>
<https://goo.gl/fQ3wqJ> <https://goo.gl/H5jBiQ> <https://goo.gl/8v95eH>
<https://goo.gl/rgt4Uo> <https://goo.gl/5nAnsW>

**Reply by andrebaltieri** [reply]
Bom dia, @laurolapajunior, como vai?

Não... é pra funcionar com páginas e componentes!
Que bom que o `dense` funcionou então... show!

**Comment by laurolapajunior** [comment]
Boa tarde, depois vou ver novamente , mas só consegui no componente,
obrigado, mas uma outra questão, você usa muito Rider, inclusive estou
usando ele, estou com dificuldade em debug no Rider, tem que fazer algo?

Em seg., 15 de jul. de 2024 07:59, Andre Baltieri ***@***.***>
escreveu:

**Reply by andrebaltieri** [reply]
No Mac ainda não tem debug no Edge, então tem que ter o Chrome instalado (Recomendo no Windows também).
Mas me conta sua dificuldade que vejo o que pode ser.

**Comment by laurolapajunior** [comment]
quando tento rodar em debug, apresenta o erro abaixo

[image: image.png]

Em ter., 16 de jul. de 2024 às 13:09, Andre Baltieri <
***@***.***> escreveu:

> No Mac ainda não tem debug no Edge, então tem que ter o Chrome instalado
> (Recomendo no Windows também).
> Mas me conta sua dificuldade que vejo o que pode ser.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/615#discussioncomment-10064468>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZUHEU3TYAGD3JEPIEE72DZMVASPAVCNFSM6AAAAABKT5HKPWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBWGQ2DMOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Lauro
Batista Lapa Junior

Analista de Sistemas
3L Soluções
***@***.***
(44) 98850-2722
<https://goo.gl/qxcoqu> <https://goo.gl/JksoVf> <https://goo.gl/dL8XRG>
<https://goo.gl/fQ3wqJ> <https://goo.gl/H5jBiQ> <https://goo.gl/8v95eH>
<https://goo.gl/rgt4Uo> <https://goo.gl/5nAnsW>

**Reply by andrebaltieri** [reply]
@laurolapajunior bom dia! A imagem não veio!

**Comment by laurolapajunior** [comment]
bom dia, estranho, para mim aparece, mandar novamente.

obrigado

Em qua., 17 de jul. de 2024 às 08:47, Andre Baltieri <
***@***.***> escreveu:

> @laurolapajunior <https://github.com/laurolapajunior> bom dia! A imagem
> não veio!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/615#discussioncomment-10072738>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZUHEUVRNTE6ROP2YPWUS3ZMZKU5AVCNFSM6AAAAABKT5HKPWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBXGI3TGOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Lauro
Batista Lapa Junior

Analista de Sistemas
3L Soluções
***@***.***
(44) 98850-2722
<https://goo.gl/qxcoqu> <https://goo.gl/JksoVf> <https://goo.gl/dL8XRG>
<https://goo.gl/fQ3wqJ> <https://goo.gl/H5jBiQ> <https://goo.gl/8v95eH>
<https://goo.gl/rgt4Uo> <https://goo.gl/5nAnsW>
